### ISSUE TITLE
Pytest

### ISSUE THREAD

**Issue by dave-donaghy** [muhammadrevo/Guess-the-Number#2]
I love pytest, and found it ridiculously easy to use once I worked out a couple of themes that it uses.

Start everything (well, file names and test method names...) with "test_", and you're away!

**Comment by muhammadrevo** [comment]
@dave-donaghy  I completely agree, let my point out that when I actually worked on this project, I have just about one week of Python knowledge. Now, I think I am a proficient Python developer haha. I can already spot so many improvements needed, which I will work on when I have time, that includes creating some unit testing.

**Comment by muhammad-amhan** [comment]
@dave-donaghy  I completely agree. Let me just point out something, when I worked on this project, I had just about 1 week worth of Python knowledge. Now I can call myself proficient Python developer haha. I already can spot so many code improvements, including writing some unit tests. I will when I have some time off.
